src/shared/publicURL.js

```diff
@@ -1,7 +1,7 @@
 const path = () => {
     var location = document.createElement("a")
     location.href = process.env.PUBLIC_URL
-    return location.pathname
+    return (process.env.PUBLIC_URL !== '') ? location.pathname : ''
```

Comment:
Is this validation really necessary?

Reply:
yes, otherwise the path is '/' and it generates errors